.circleci/example.sh

```diff
@@ -52,16 +52,29 @@ go() {
   fi
   echo "[ \"${STATUS}\", \"${CONN}.${RANK}\" ]" >/tmp/workspace/record/"${THIS}"
 }
-for WHICH in $(find . -maxdepth 1 -type d | sed 'sX./XX' | sort | tail -n +2 | awk "NR % ${SIZE} == ${RANK}") ; do
-  KIDS=$((KIDS + 1))
-  go "${WHICH}" &
-done
 
-while true; do
-  wait -n || true
-  KIDS=$((KIDS - 1))
-  if [ $KIDS -eq 0 ]; then break; fi
-done
+EXS="$(find . -maxdepth 1 -type d | sed 'sX./XX' | sort | tail -n +2 | awk "NR % ${SIZE} == ${RANK}")"
+
+case "${CONN}" in
+  # XXX Conflux' transparent nonce selection currently isn't robust enough to
```

Comment:
This is too slow though... you should instead make it so they are forked but have their own devnets